app/views/users/profile.html.erb

```diff
@@ -72,6 +84,24 @@
     <% end %>
   </p>
   <hr />
+    <div class="row">
+      <div class="col-md-3 col-md-offset-1">
+        <h4>Followers</h4>
```

Comment:
In the way you've nested the rows here, if it becomes single-column (as on a mobile phone) they won't associate or stack correctly, I think. Can you try just one "row" and within each column, have a `<br />` between the label "Following" and the count? That way the columns will stack instead of the rows. Try the "device toolbar" in Chrome to simulate the screen width of, say, an iPhone or Android phone.